src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/OverloadMethodsDeclarationOrder/context/defaultContextConfig.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <module name="TreeWalker">
+        <module name="OverloadMethodsDeclarationOrder"/>
+
+        <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchXpathFilter">
+            <property name="file" value="${tp}/defaultContext.patch" />
+            <property name="strategy" value="context" />
+            <property name="neverSuppressedChecks" value="OverloadMethodsDeclarationOrderCheck" />
```

Comment:
Why do you put it under this group? Can we use checkNamesForContextStrategyByTokenOrParentSet option instead?

Reply:
yes, it's ok, I use `checkNamesForContextStrategyByTokenOrParentSet` now.